src/Assignments/SimpleProblemPage.tsx

```diff
@@ -379,12 +418,15 @@ export const SimpleProblemPage: React.FC<SimpleProblemPageProps> = () => {
                                                 New Version
                                             </Button>
                                         }
-                                        <Button variant='danger'
-                                            tabIndex={0}
-                                            onClick={() => confirmEndVersion()}
-                                        >
-                                            End Exam
-                                        </Button>
+                                        {(attemptsRemaining!==topic.topicAssessmentInfo?.maxGradedAttemptsPerVersion) &&
```

Comment:
Shouldn't use strict equals, should be
```
attemptsUsed < maxAttempts
```
in case there is a bug and they somehow get over the max attempts, then we want some damage control